fe/fe-core/src/main/java/org/apache/doris/analysis/ModifyTablePropertiesClause.java

```diff
@@ -350,6 +350,19 @@ public void analyze(Analyzer analyzer) throws AnalysisException {
             // do nothing, will be analyzed when creating alter job
         } else if (properties.containsKey(PropertyAnalyzer.PROPERTIES_ROW_STORE_COLUMNS)) {
             // do nothing, will be analyzed when creating alter job
+        } else if (properties.containsKey(PropertyAnalyzer.PROPERTIES_AUTO_ANALYZE_POLICY)) {
+            String analyzePolicy = properties.getOrDefault(PropertyAnalyzer.PROPERTIES_AUTO_ANALYZE_POLICY, "");
+            if (analyzePolicy != null
+                    && !analyzePolicy.equals(PropertyAnalyzer.ENABLE_AUTO_ANALYZE_POLICY)
+                    && !analyzePolicy.equals(PropertyAnalyzer.DISABLE_AUTO_ANALYZE_POLICY)
+                    && !analyzePolicy.equals(PropertyAnalyzer.NONE_AUTO_ANALYZE_POLICY)) {
+                throw new AnalysisException(
+                    "Table auto analyze policy only support for " + PropertyAnalyzer.ENABLE_AUTO_ANALYZE_POLICY
+                        + " or " + PropertyAnalyzer.DISABLE_AUTO_ANALYZE_POLICY
+                        + " or " + PropertyAnalyzer.NONE_AUTO_ANALYZE_POLICY);
```

Comment:
I think better change "none" to "base_on_system" or "base_on_config".
"none" is meaningless

Reply:
changed to "base_on_catalog"